api/swagger.yml

```
@@ -386,6 +386,36 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Diff"
+    OtfDiff:
```

a suggestion - up to you, I think that it makes it clearer that this object includes a diff record rather than the whole diff
```suggestion
    OtfDiffRecord:
```